application/api/capacityauth/admin.py

```diff
@@ -0,0 +1,46 @@
+# organizations/admin.py
+import typing
+from django.forms import ModelForm
+from django.contrib import admin
+from django.db import models
+from django.http.request import HttpRequest
+from rest_framework_api_key.admin import APIKeyModelAdmin
+from .models import DosUserAPIKey
+
+class DosUserAPIKeyAdminForm(ModelForm):
```

Comment:
So the ModelForm needs a Model instance to work properly, right..? See: https://docs.djangoproject.com/en/3.0/topics/forms/modelforms/ This is usually done via the `Meta` class into which you can say the model that's being used as the basis for the form and the fields which you want to use from the model within the form you're specifying. E.g.:

```
class Meta:
    model = DosUserAPIKey
    fields = ["foo", "bar", "baz", ]  # Only these fields will be displayed in the form.
```

The docs I link to above tell you how this all works.